Aptoide/aptoide-client-v8
# [ASV-1265] Promotions Visual Bugs

**What does this PR do?**

   Fixed some layout problems like promotions view Toolbar title broken in other languages, Claim Dialog Title margins & button width's.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] claim_promotion_captcha_view.xml

**How should this be manually tested?**

  Check the app, for example in Portuguese, and check if the problems reported on the screens attached on the ticket are fixed or not.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1265](https://aptoide.atlassian.net/browse/ASV-1265)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.) 




**Code Review Checklist**
